src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -19,6 +19,9 @@ The .NET Foundation licenses this file to you under the MIT license.
     <CppLibCreator>lib</CppLibCreator>
     <FullRuntimeName>Runtime</FullRuntimeName>
     <FullRuntimeName Condition="'$(ServerGarbageCollection)' != ''">Runtime.ServerGC</FullRuntimeName>
+    <ApplicationEntryName Condition="'$(ApplicationEntryName)' == ''">wmainCRTStartup</ApplicationEntryName>
```

Comment:
Could you name these `SubSystem` and `EntryPointSymbol`? It's how they're named in Visual C++ projects. In the absence of other reasons, it feels better to stick with naming that already exists.

Reply:
`EntryPointSymbol` make better sense to me. Reason why I prepend `Linker` to `SubSystem` is because I was not sure that this is not enough generic name, that would not be used in other applications.